packages/grpc-js/src/server-call.ts

```diff
@@ -86,25 +89,22 @@ export type ServerSurfaceCall = {
 export type ServerUnaryCall<RequestType, ResponseType> = ServerSurfaceCall & {
   request: RequestType;
 };
-export type ServerReadableStream<
-  RequestType,
-  ResponseType
-> = ServerSurfaceCall & ObjectReadable<RequestType>;
-export type ServerWritableStream<
-  RequestType,
-  ResponseType
-> = ServerSurfaceCall &
-  ObjectWritable<ResponseType> & {
-    request: RequestType;
-    end: (metadata?: Metadata) => void;
-  };
+export type ServerReadableStream<RequestType, ResponseType> =
```

Comment:
this got reformatted, I think its prettier from the project that forced the styling